conf/messages.en

```diff
@@ -137,7 +137,7 @@ amendAreYouSure.change.hidden = AmendAreYouSure
 
 amend.confirmation.title = Your return will be amended
 amend.confirmation.submitted = Your return will be amended
-amend.confirmation.panel = Your reference number for this amendment is PPT12345654321
+amend.confirmation.panel =
```

Comment:
Its this intentionally empty?

Reply:
Yeah, I think blank is better than faked for now?